libs/utils/src/hooks/useGetMovies.tsx

```diff
@@ -0,0 +1,30 @@
+import { Movie } from '@form-exercise/data/interface';
+import axios, { AxiosError } from 'axios';
+import { useQuery, UseQueryResult } from 'react-query';
+
+interface UseGetMovies {
+  searchBy: string;
+}
+
+interface Movies {
+  Reponse: string;
+  Search: Movie[];
+  totalResults: string;
+}
+
+export function useGetMovies({
+  searchBy,
+}: UseGetMovies): UseQueryResult<Movies, AxiosError<unknown, any>> {
+  const getMovies = async (): Promise<Movies> => {
+    const { data } = await axios(
+      `http://www.omdbapi.com/?apikey=${process.env['NX_OMDB_API_KEY']}&s=${searchBy}&type=movie`
```

Comment:
can use this directly
```ts
process.env. NX_OMDB_API_KEY
```

also, with axios you can pass parameters easily like this:
```ts
const { data } = await axios('http://www.omdbapi.com', { params: { type: 'movie', apiKey: process.env. NX_OMDB_API_KEY, s: searchBy } })
```